.github/workflows/agent-build.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     strategy:
       matrix:
         os: [linux]
-        php_ver: ['7.0', '7.1-zts', '7.2', '7.3', '7.4', '8.0', '8.0-zts', '8.1', '8.1-zts']
+        php_ver: ['7.0', '7.1-zts', '7.2', '7.3', '7.4', '8.0', '8.0-zts', '8.1', '8.1-zts', '8.2']
         arch: ['x64', 'x86']
```

Comment:
You can just remove 'x86' from this and probably lose all the `excludes`.

Reply:
Agreed, but probably best to handle that in a separate PR

Reply:
I've done it in the oapi branch already so it will get picked up.